lib/migration.pm

```diff
@@ -207,4 +209,13 @@ sub set_scc_proxy_url {
     save_screenshot;
 }
 
+# jira feature https://jira.suse.com/browse/SLE-21973
+# zypper single transaction performance enhancement (RFE)
```

Comment:
@hjluo These two lines too :)

Reply:
can you please take a look again. thanks @foursixnine